hack/set-openshift-security-context/main.go

```diff
@@ -0,0 +1,133 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"strconv"
+
+	"github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+
+	"github.com/open-telemetry/opentelemetry-operator/pkg/autodetect"
+	"github.com/open-telemetry/opentelemetry-operator/pkg/platform"
+)
+
+func parseId(namespace *corev1.Namespace, annotation string) int64 {
+	raw := namespace.GetAnnotations()["openshift.io/sa.scc.supplemental-groups"]
+	if raw == "" {
+		fmt.Println("The annotation ", annotation, " is not present")
+		os.Exit(1)
+	}
+
+	lowBound := strings.Split(raw, "/")[0]
+	id, err := strconv.ParseInt(lowBound, 0, 64)
+	if err != nil {
+		fmt.Println("It was not possible to convert the number to int64: ", lowBound)
+		os.Exit(1)
```

Comment:
To keep good practices I would prefer to return an error and exit only in the main function